Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/CreditCardBottomSheetViewController.swift

```diff
@@ -172,6 +172,7 @@ class CreditCardBottomSheetViewController: UIViewController, UITableViewDelegate
         let contentViewWidth = UX.contentViewWidth > view.frame.width ? view.frame.width - UX.containerPadding : UX.contentViewWidth
         contentWidthConstraint = contentView.widthAnchor.constraint(equalToConstant: contentViewWidth)
         contentWidthConstraint.priority = UILayoutPriority(999)
+        cardTableView.isScrollEnabled = viewModel.creditCards?.count ?? 0 > 6
```

Comment:
How is this behaving when using large dynamic type? I am worried that cards can not be scrolled then. Shouldn't we solve this in a way that scrolls when the content doesn't fit instead of using a number of cards?

Reply:
good catch, modified accordingly